### ISSUE TITLE
Hair color changed

### ISSUE THREAD

**Issue by JoeCleary** [adamburgess/celeste-high-frame-rate#2]
![image](https://user-images.githubusercontent.com/18452763/102461851-022f1e80-4017-11eb-83e8-1df98d158d04.png)

not really an issue but just curious why the hair color is different after a dash, is it due to the mod?

**Comment by doughn0** [comment]
I would guess it was put there by the dev to make detecting stuff like this possible in speedrun videos.

**Comment by JoeCleary** [comment]
That's what I'm thinking too now, it went away when I used vsync, limiting it to 115fps for my display.
One thing that may deserve a separate issue is that some physics are ever-so-slightly different. I started using this after I finished the main levels so I don't know if this caused a difference there, but I ran into an issue in the Golden Ridge C side, with the green bubbles not giving me either the correct distance or speed so I couldn't complete the level unless my timing was perfect. 

I'm guessing it was due to some floating point error or something when running at like 1000 fps.
Really doesn't seem worth looking into though, just tell people to use Vsync or something to limit fps.
